<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tools::SelectionViewInfo</class>
 <widget class="QDockWidget" name="Tools::SelectionViewInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Selection Info</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <action name="actionSelection_Info">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Selection &amp;Info</string>
   </property>
   <property name="toolTip">
    <string>Show information about the current selection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+I</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
